src/facets/FundsFacet.sol

```diff
@@ -148,13 +129,7 @@ contract FundsFacet is RoleCheck, PausableCheck, ERC1155SupplyUpgradeable, IFund
         require(LibClients._isProjectActive(projectId), LibErrors.ProjectInactive());
 
         LibFunds.FundsStorage storage sF = LibFunds._getFundsStorage();
-        uint256 newTotalAssets;
-        if (sF.lastTotalAssetsTimestamp + sF.lastTotalAssetsUpdateInterval < block.timestamp) {
```

Comment:
Propose to remove it. On ERC4626 it is better to not use stale data and simplify new view accounting methods on FundsFacet.